integration-cli/docker_api_set_unix_test.go

```diff
@@ -0,0 +1,43 @@
+// +build !windows
```

Comment:
I did that before, checked with `testRequires(c, DaemonIsLinux)` for all cases, then @jhowardmsft suggested we move these tests to `_unix` test file, I think all `_unix` test files are built with !windows.
ping @jhowardmsft what would you prefer?